apps/balancer-tools/src/app/apr/pool/(components)/HistoricalAPRChart.tsx

```diff
@@ -26,23 +26,23 @@ export default async function HistoricalAPRChart({
   const aprPerRoundCords = Object.entries(results.perRound).reduce(
     (cords, [_, result]) => {
       cords.x.push(getRoundName(result.roundId));
-      cords.y.push(result.apr);
+      cords.y.push(result.apr.breakdown.veBAL);
```

Comment:
could we also add total and veBAL as separate series in this plot? This would already prepare us for when the total considers other APR sources as well.

Reply:
sure! I was going to add this only when we had the fees APR so we don't confuse the potential users